data/human/human missions.txt

```diff
@@ -6303,9 +6303,9 @@ mission "Blind Man from Martini"
 			`	With that, he grabs his cane and his luggage, and follows a hospital attendant toward his daughter's room, leaving you in the reception.`
 
 mission "Care Package to South 1"
-	minor
```

Comment:
But that was right. :(

Reply:
It was? I better update my style guide...